webui/module/Director/src/Director/Model/DirectorModel.php

```diff
@@ -86,6 +86,38 @@ public function getDirectorStatus(&$bsock=null)
       }
    }
 
+   /**
+    * Get Director status subscription
+    *
+    * @param $bsock
+    *
+    * @return string
+    */
+   public function getStatusSubscriptionDetail(&$bsock=null)
+   {
+      if(!isset($bsock)) {
+         throw new \Exception('Missing argument.');
+      }
+      $result = $bsock->send_command("status subscription detail", 0);
+      return $result;
+   }
+
+   public function getStatusSubscriptionAll(&$bsock=null)
```

Comment:
Is it possible to unify the two functions? Like:
`getStatusSubscription(&$bsock=null, option="", json=0)`
with  `option = "" | "all" | detail`
If not, the function names should indicate, what the function does. Both function have nearly the same name, but one is returning text, the other a json dictionary.

Reply:
Unified, see https://github.com/bareos/bareos/pull/1280/commits/936698c227172bcd303b900e91dd426978b33cb7 .

Reply:
Updated: https://github.com/bareos/bareos/pull/1280/commits/bd89100dfd7efe20c322ba079f6c7f47d69fb3b7